index.spec.js

```diff
@@ -1,38 +1,47 @@
 const { minVal, fizzBuzz, timesTwo, returnObjects, removeBMW } = require('./index');
-
 test('passes when value is NaN', () => {
     expect(NaN).toBeNaN();
     expect(1).not.toBeNaN();
     expect(typeof NaN).toBe('number');
 });
 
 describe('Testing function minVal()', () => {
-  
-    it('Should throw an error if a parameter is not valid', () => {
-     expect(()=> {
-         minVal([])
-     }).toThrow("enter a valid value"); 
+
+    it('Should throw an error if perameter empty', () => {
+        expect(() => {
+          minVal();
+        }).toThrow("Error: Parameter is empty");
     });
-    it('Shoul return a value of data type number', () => {
+
+    it('Should throw an error if array empty', () => {
+        expect(() => {
+          minVal([]);
+        }).toThrow("Error: Array is empty");
+    });
+
+    it('Should return a value of data type number', () => {
         expect(typeof minVal([1,2,34,0,-12])).toBe('number');
         expect(minVal([1,2,34,0,-12])).not.toBeInstanceOf(Object);
     });
-  
+
     it('Should get a minimal value from an array', () => {
-        expect(minVal([1,2,34,0,-12])).toBe(-12);
+        expect(minVal([2, -12])).toBe(-12);
```

Comment:
I am not sure this improves the readability of tests. Was that the reason for this change?

Reply:
Resolved ,given better test data